harness/determined/cli/workspace.py

```diff
@@ -149,6 +149,26 @@ def list_workspace_projects(args: Namespace) -> None:
         render.tabulate_or_csv(PROJECT_HEADERS, values, False)
 
 
+@authentication.required
+def list_pools(args: Namespace) -> None:
+    session = cli.setup_session(args)
+    w = workspace_by_name(session, args.workspace_name)
+    resp = bindings.get_ListRPsBoundToWorkspace(session, workspaceId=w.id)
+
+    pools_str = ""
+    if resp.resourcePools is None:
```

Comment:
I am far from expert in the CLI, so please trust this only partially: I think it's more common to print an empty table in cases like this than to print an error. There's also a `--json` flag that's sometimes supported. In that was passed (and implemented), having a parseable response would be nice, and if that was the behavior, then maybe it'd be nice to have analogous behavior with/without the flag.

But there are lots of conditionals in that justification and if you prefer the "exceptional" style here, I won't push back.

Also I guess while on the subject, want to implement a `--json`? (not required -- its support isn't universal)

Reply:
I'll do this alongside the refactor to use SDK functions. I'm thinking if I can get to it tonight, it'll happen now, but otherwise I'll do it before release.